gatovid/api/data.py

```diff
@@ -32,3 +37,56 @@ def test():
             "stats": str(user.stats),
         },
     }
+
+
+@mod.route("/login", methods=["GET", "POST"])
+def login():
+    email = request.args.get("email")
+    password = request.args.get("password")
+
+    if None in (email, password):
+        return {
+            "error": "Parámetro vacío",
+        }
+
+    # Comprobamos si existe un usuario con ese email
+    user = User.query.get(email)
+    if not user:
+        return {
+            "error": "El usuario no existe",
+        }
+
+    # Comprobamos si los hashes coinciden
+    if user.check_password(password) is False:
```

Comment:
`if not user.check_password`

Reply:
Es que me parecía más legible así jajaja. De la otra forma si lo lees es "si no checkeamos la contraseña".

Reply:
supongo que puedo cambiarlo, da igual

Reply:
si lo ves raro le puedes cambiar el nombre a `password_matches` por ejemplo

Reply:
da igual, ya esta como has pedido